src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,15 @@
+$block-size: 100px;
+$block-padding: 10px;
```

Comment:
If you have block-size that equals to width and height of the block than it's logical to suppose that block-padding is the padding of this block. But you use it as padding of calendar. You should probably name it calendar-padding than.